examples/Batch/WordCountFSharp/WordCountExample.fs

```diff
@@ -0,0 +1,34 @@
+﻿open Microsoft.Spark.CSharp.Core
+open Microsoft.Spark.CSharp.Services
+open System.Reflection
+open System.Collections.Generic
+
+[<EntryPoint>]
+let main = function
```

Comment:
This use of ``function`` is not particularly idiomatic - in general I think it's better to avoid ``function`` in beginner samples and just build confidence/knowledge with an explicit argument and a ``match``